pkg/cmd/issue/create/create.go

```diff
@@ -194,6 +194,14 @@ func createRun(opts *CreateOptions) (err error) {
 				return
 			}
 
+			// overwrites previous templateContent
+			var templates []api.IssueTemplate
+			templates, err = api.IssueTemplates(apiClient, repo)
+			templateContent = templates[0].Body
+			if err != nil {
+				return
+			}
```

Comment:
From the discussion in the issue I think this API call should replace the call to `prShared.FindTemplates` on line 108. 

I am going to cc @mislav since it was his his comment about using the API both when `-R` is provided and when it is not which makes me think that is the desired approach.

Reply:
Thanks @samcoe for the initial review! You are right that this call needs to happen earlier.

@divyaramanathan You will want to invoke the new API logic just before `TemplateSurvey()`, since that is where the user will be presented with a list of templates to choose between. Also, since the new API approach replaces looking up templates from the filesystem, you can remove the old filesystem lookups from this file.